addons/mail/static/src/models/rtc_controller.js

```diff
@@ -74,6 +74,17 @@ registerModel({
                 startWithVideo: true,
             });
         },
+        /**
+         * @private
+         * @returns {string}
+         */
+        _computeHeadphoneTitle() {
+            if (this.messaging.rtc.currentRtcSession.isDeaf) {
```

Comment:
You should ensure that `currentRtcSession` is defined before trying to access `isDeaf`.

Reply:
Template has no guard, so I think it never happens having `RtcController` and no `currentRtcSession`?

Reply:
I've tested it and it crashes

Reply:
Oh ok, then I'll change it

Reply:
Changes done